config/backstop/system.views.xml

```diff
@@ -0,0 +1,685 @@
+            resourcelabels="false">
+            <desc><![CDATA[The DataType Object form.]]></desc>
+            <altviews>
+                <altview name="DataType Edit View" viewdef="DataType" mode="view" validated="false"/>
+                <altview name="DataType Edit" viewdef="DataType" mode="edit" validated="true" default="true"/>
+            </altviews>
+        </view>
+        
+        <view name="UserGroup"
+            class="edu.ku.brc.specify.datamodel.SpPrincipal"
+            resourcelabels="false">
+            <desc><![CDATA[User Group form.]]></desc>
+            <altviews defaultmode="view">
+                <altview name="User Group View"        title="User Group"        viewdef="User Group"        mode="view" validated="false"/>
+                <altview name="User Group Edit"        title="User Group"        viewdef="User Group"        mode="edit" validated="true" default="true"/>
+            </altviews>
+        </view>
+
+        <view name="User"
+            class="edu.ku.brc.specify.datamodel.SpecifyUser"
+            busrules="edu.ku.brc.specify.datamodel.busrules.SpecifyUserBusRules"
+            resourcelabels="false">
+            <desc><![CDATA[User form.]]></desc>
+            <altviews defaultmode="view">
+                <altview name="User View"        title="User"        viewdef="User"        mode="view" validated="false"/>
+                <altview name="User Edit"        title="User"        viewdef="User"        mode="edit" validated="true" default="true"/>
+            </altviews>
+        </view>
+
+        <view name="CatalogNumberingScheme"
+            class="edu.ku.brc.specify.datamodel.CatalogNumberingScheme"
+            busrules="edu.ku.brc.specify.datamodel.busrules.CatalogNumberingSchemeBusRules"
+            resourcelabels="false">
+            <desc><![CDATA[The CatalogNumberingScheme Object form.]]></desc>
+            <altviews>
+                <altview name="CatalogNumberingScheme Edit View" viewdef="CatalogNumberingScheme" mode="view" validated="false"/>
+                <altview name="CatalogNumberingScheme Edit" viewdef="CatalogNumberingScheme" mode="edit" validated="true" default="true"/>
+            </altviews>
+        </view>
+        
+        <!--  <view name="ViewProps"
+            class="edu.ku.brc.specify.datamodel.SpView"
+            resourcelabels="false">
+            <desc><![CDATA[]]></desc>
+            <altviews>
+                <altview name="View Edit" viewdef="ViewProps" mode="edit" validated="true"/>
+            </altviews>
+        </view> -->
+        
+        <view name="ExcelExportInfo"
+            class="java.util.Hashtable"
+            resourcelabels="false">
+            <desc><![CDATA[]]></desc>
+            <altviews>
+                <altview name="ExcelExportInfo Edit" viewdef="ExcelExportInfo" mode="edit" validated="true"/>
+            </altviews>
+        </view>
+        
+        <view name="ChangePassword"
```

Comment:
remove.
search for all views that don't have a `class="edu.ku.brc.specify.datamodel.*` attribute and consider removing them